README.md

```diff
@@ -29,15 +29,27 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. Describe the biggest difference between `.forEach` & `.map`.
+ 
+    Map is an higher order function
 
```

Comment:
.forEach() - it calls a provided function for each element in the array , it does't actually return anything . This allows to mutate the the current array

.map() -creates a new array with the result , it calls a provided function on each element of the array and it will always returs a new array of the same size